tools/validators/instance_validator/validate/subscriber.py

```diff
@@ -31,10 +31,9 @@ class Subscriber(object):
     service_account_info: service account information from the GCP project.
   """
 
-  def __init__(self, subscription_name, service_account_info_json_file):
+  def __init__(self, subscription_name, service_account_info_json_file = None):
```

Comment:
can you please update this to : `service_account_info_json_file : Optional [str] =None`